worker/process-image.js

```diff
@@ -10,6 +12,8 @@ var applyGdalinfo = require('oam-meta-generator/lib/apply-gdalinfo');
 var sharp = require('sharp');
 var log = require('./log');
 var config = require('../config');
+// Must be modified to match the local path to the gdal_translate executable
+var translateExe = '/Library/Frameworks/GDAL.framework/Versions/2.1/Programs/gdal_translate';
```

Comment:
Instead of hardcoding the path in this file we should use the config file, where the path is set through a ENV variable